__tests__/app.test.js

```diff
@@ -44,3 +56,47 @@ describe("GET /api/categories", () => {
         .expect(404)
     })
 })
+
+
+describe("GET /api/reviews", () => {
+    test("Responds with an array of review objects, each with these properties: owner, title, review_id, category, review_img_url,  created_at, votes, designer, comment_count.", () => {
+    return request(app)
+        .get('/api/reviews')
+        .expect(200)
+        .then(( {body} ) => {
+            const {reviews} = body;
+            expect(reviews).toBeInstanceOf(Array);
+            expect(reviews).toHaveLength(13);
+            reviews.forEach((review) => {
+                expect(review).toEqual(
+                    expect.objectContaining({
+                        owner: expect.any(String),
+                        title: expect.any(String),
+                        review_id: expect.any(Number),
+                        category: expect.any(String),
+                        review_img_url: expect.any(String),
+                        created_at: expect.any(String),
+                        votes: expect.any(Number),
+                        designer: expect.any(String),
+                        comment_count: expect.any(String)
+                    })
+                )
+            })
+        })
+    })
+
+    test("Responds with a key of reviews sorted by date (created_at) in descending order.", () => {
+        return request(app)
+            .get('/api/reviews')
+            .expect(200)
+            .then((res) => {
+                expect(res.body.reviews).toBeSortedBy('created_at', {descending: true});
+            })
+    })
+
+    test("GET /api/not-route, returns 404", () => {
```

Comment:
It would be nice to break the invalid paths out to a seperate describe block as they arn't direcetly related to the endpoints functionality.